nested/controller.json

```diff
@@ -0,0 +1,234 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "moodleCommon": {
+            "metadata": {
+                "description": "Common Moodle values"
+            },
+            "type": "object"
+        },
+        "blobStorageAccountKey": {
+            "metadata": {
+                "description": "Key used to create a storage container"
+            },
+            "type": "string"
```

Comment:
secureString

Reply:
Done

Reply:
More episodes... I realized that the keys don't have to be passed around, so I got rid of the outputs and params. Initially I put listKeys() in variables, which is not allowed by ARM. After all, I substituted the whole expressions (including listKeys()) for variables, so now there's no more string secrets being passed around...